src/components/Articles/ArticleList.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react"
+import axios from "axios"
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([])
+  const apiUrl = "https://backend-project-d9id.onrender.com/api/articles"
+
+  useEffect(() => {
+    const fetchArticles = async () => {
```

Comment:
It would be nice if you could store your functions for your axios requests in a utils folder that you can then export/import as needed. 

If you have all your axios requests in one file you can then use axios.create to generate the base URL that you can use in each function.